czkawka_core/src/similar_videos.rs

```diff
@@ -243,106 +243,124 @@ impl SimilarVideos {
+                            atomic_file_counter.fetch_add(1, Ordering::Relaxed);
+
+                            let file_name_lowercase: String = match entry_data.file_name().into_string() {
+                                Ok(t) => t,
+                                Err(_inspected) => {
+                                    println!("File {:?} has not valid UTF-8 name", entry_data);
+                                    continue 'dir;
+                                }
+                            }
+                            .to_lowercase();
+
+                            if !self.allowed_extensions.matches_filename(&file_name_lowercase) {
+                                continue 'dir;
+                            }
+
+                            if ![".mp4", ".mpv", ".flv", ".mp4a", ".webm", ".mpg", ".mp2", ".mpeg", ".m4p", ".m4v", ".avi", ".wmv", ".qt", ".mov", ".swf", ".mkv"]
+                                .iter()
+                                .any(|e| file_name_lowercase.ends_with(e))
+                            {
+                                continue 'dir;
+                            }
+
+                            // Checking files
+                            if (self.minimal_file_size..=self.maximal_file_size).contains(&metadata.len()) {
+                                let current_file_name = current_folder.join(entry_data.file_name());
+                                if self.excluded_items.is_excluded(&current_file_name) {
+                                    continue 'dir;
+                                }
+
+                                let fe: FileEntry = FileEntry {
+                                    path: current_file_name.clone(),
+                                    size: metadata.len(),
+                                    modified_date: match metadata.modified() {
+                                        Ok(t) => match t.duration_since(UNIX_EPOCH) {
+                                            Ok(d) => d.as_secs(),
+                                            Err(_inspected) => {
+                                                warnings.push(format!("File {} seems to be modified before Unix Epoch.", current_file_name.display()));
+                                                0
+                                            }
+                                        },
+                                        Err(e) => {
+                                            warnings.push(format!("Unable to get modification date from file {}, reason {}", current_file_name.display(), e));
+                                            0
+                                        }
+                                    },
+                                    vhash: Default::default(),
+                                    error: "".to_string(),
+                                };
+
+                                fe_result.push((entry_data.file_name().to_string_lossy().to_string(), fe));
```

Comment:
was full path with `current_folder.join(entry_data.file_name())`, now is `entry_data.file_name()`, why the change and what does that mean?